tests/e2e_regression/run.sh

```diff
@@ -17,12 +20,39 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 # The hostname of the API server to test
 hostname="http://localhost:8008"
 
+# The command to invoke psql (complete with connection params)
+psql="$(make --dry-run --no-print-directory psql)"
+
 # The directory to store the actual responses in
 outDir="$SCRIPT_DIR/actual"
 mkdir -p "$outDir"
 rm "$outDir"/* || true
 
+# Each test case is a pair of (name, SQL query or URL).
+# For SQL queries, the regression-tested output is the result of the query against the indexer DB.
+# For URLs, the regression-tested output is the full HTTP response.
 testCases=(
+  ## Consensus.
+  'db:allowances               select * from chain.allowances order by beneficiary, owner'
```

Comment:
Seems that Github artifacts doesn't like colons in the path name. 

> Error: Artifact path is not valid: /db:allowances.csv. Contains the following character:  Colon :

https://github.com/oasisprotocol/nexus/actions/runs/6305740450/job/17119705569?pr=525